build-tests/node-library-build-eslint-test/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "lib/index.js",
   "license": "MIT",
   "scripts": {
-    "build": "gulp test --clean"
+    "build": "gulp test --clean",
+    "lint": "node ../../common/temp/node_modules/eslint/bin/eslint -f unix \"src/**/*.{ts,tsx}\""
```

Comment:
Just make eslint a dependency.

Reply:
I didn't want to do that, since part of what we are testing here is that RSC resolves ESLint correctly without a direct dependency. Whereas the "lint" script is just for debugging.